pkg/models/system_intake_relation.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "github.com/google/uuid"
+
+// CreateSystemIntakeContractNumbersLink is the insertion type for linking a system intake to
+// contract number(s)
+type CreateSystemIntakeContractNumbersLink struct {
+	ID             uuid.UUID `db:"id"`
+	IntakeID       uuid.UUID `db:"intake_id"`
+	ContractNumber string    `db:"contract_number"`
+	CreatedBy      string    `db:"created_by"`
+	ModifiedBy     string    `db:"modified_by"`
+}
```

Comment:
I think we should probably:
- Rename this struct to just `SystemIntakeContractNumber`
- Add `BaseStruct`

then you can use this type not just for `create` operations, but also `SELECT`! (I'll leave a top-level PR comment about that!)

Reply:
Nice! Love that! Thank you - updated!